### PR TITLE
Update math.vector.ts

### PR DESCRIPTION
**Pull request by aWeirdo**
Adds static "Center**ToRef**" for Vector2/3/4

### CODE REVIEW

**Comment by deltakosh** [comment]
We are in code freeze so we can only merge bug fixes

I will merge this PR as soon as 4.2 will be out :)

File: src/Maths/math.vector.ts [hunk 1]
```diff
@@ -656,6 +656,17 @@ export class Vector2 {
         return center;
     }
 
+    /**
```
Comment (deltakosh) [inline src/Maths/math.vector.ts:659]: can you fix conflicts and make sure that CEnter is using CenterToRef?

Comment (aWeirdo) [reply src/Maths/math.vector.ts:659]: Will be done by monday :)

**Comment by aWeirdo** [comment]
Github giving problems :)
https://github.com/BabylonJS/Babylon.js/pull/9406

**Pull request closed without merging**